extensions/rich_text_components/Collapsible/directives/CollapsibleDirective.js

```diff
@@ -20,12 +20,14 @@
  * followed by the name of the arg.
  */
 oppia.directive('oppiaNoninteractiveCollapsible', [
-  '$rootScope', '$sce', 'HtmlEscaperService',
-  function($rootScope, $sce, HtmlEscaperService) {
+  '$rootScope', '$sce', 'HtmlEscaperService', 'UrlInterpolationService',
+  function($rootScope, $sce, HtmlEscaperService, UrlInterpolationService) {
     return {
       restrict: 'E',
       scope: {},
-      templateUrl: 'richTextComponent/Collapsible',
+      templateUrl: UrlInterpolationService.getExtensionResourceUrl(
```

Comment:
Can we do: UrlInterpolationService.getRteDirectiveHtmlUrl('Collapsible')? Ditto elsewhere.

Reply:
How would this work if there were multiple directives for a component?

Reply:
That ... is an excellent question. I thought about maybe doing UrlInterpolationService.getRteDirectiveHtmlUrl('Collapsible', 'rte_collapsible_directive.html') or similar, but then -- it's not clear it's worth it, really. So I'm happy with what you  have, too.

Another option is to assume for now that there won't be multiple directives for a component, and then extend later if this turns out not to be the case. (With 6-7 components so far, we haven't ever run into that, so that's a data point FWIW.) But I'm happy either way.

Reply:
I didn't change it.